### ISSUE TITLE
Failed to spawn command pylint. Make sure pylint is install and on your PATH

### ISSUE THREAD

**Issue by magomez96** [AtomLinter/linter-pylint#100]
![capture](https://cloud.githubusercontent.com/assets/5390341/11718788/af5f4f28-9f25-11e5-9fa4-2eb3983b8135.JPG)
I'm using Atom 1.3.0, Linter 1.11.3, Python 2.7.11, Linter-pylint 1.2.0 on Windows 7 Pro X64. I can run pylint from the command line by simply typing pylint

**Comment by Arcanemagus** [comment]
Did you customize the location of `pylint` at all? It almost looks like there is a space or some other unrenderable character at the end there.

**Comment by magomez96** [comment]
No, I did not.

**Comment by Arcanemagus** [comment]
If you type `where.exe pylint` in a command prompt what does it return?

**Comment by magomez96** [comment]
C:\Python27\Scripts\pylint.bat
C:\Python27\Scripts\pylint.exe

**Comment by Arcanemagus** [comment]
Hmm, that's exactly what it returns on my system...

Can you put a breakpoint at [line 97](https://github.com/AtomLinter/linter-pylint/blob/v1.2.0/lib/main.coffee#L97) and see what the output of this is:
```javascript
JSON.stringify(executable)
```

**Comment by magomez96** [comment]
I'm getting the same ENOENT error when I try to start node-debugger

**Comment by Arcanemagus** [comment]
You should be able to place the breakpoint from the developer tools.

**Comment by magomez96** [comment]
executable is not defined

**Comment by Arcanemagus** [comment]
What does `atom.config.get('linter-pylint.executable')` return?

**Comment by Arcanemagus** [comment]
Closing due to a lack of response. If you are still having problems feel free to post further!

**Comment by gregorylegarec** [comment]
I get this error too and after an hour of investigation, I found out that the EOENT error may also happen when the working directory does not exist on current project.

Mine is configured to be `%p/api` as most of the projects I am working on are built this way. But when I was coding simple scripts I get this error.

So, I fixed by replacing this line https://github.com/AtomLinter/linter-pylint/blob/master/lib/main.js#L116 with: 
```js
const configuredCwd = fixPathString(
  this.workingDirectory,
  fileDir,
  projectDir
);
const cwd = fs.existsSync(configuredCwd) ? configuredCwd : projectDir;
```
